src/content/docs/en/reference/api-reference.mdx

```diff
@@ -1490,18 +1696,25 @@ export function onRequest (context, next) {
 
 ### `next()`
 
-A function that intercepts (reads and modifies) the `Response` of a `Request` or calls the "next" middleware in the chain and returns a `Response`. For example, this function could modify the HTML body of a response.
+<p>
 
-This is an optional argument of `onRequest()`, and may provide the required `Response` returned by the middleware.
+**Type:** `(rewritePayload?: string | URL | Request) => Promise<Response>`<br />
+<Since v="4.13.0" />
```

Comment:
This one is a bit tricky. There has _always_ been a `next()` since middleware was first added, but the extra `rewritePayload` argument you can pass to it is new since `4.13.0` not quite sure how best to handle this situation. Might make most sense to move the mention inline with the final part about the `rewritePayload` instead of including it here?

Reply:
Ah this one is my fault! It's [currently as you describe](https://docs.astro.build/en/reference/api-reference/#next), I haven't checked the history of the argument so I've pulled the information up so it's like the other sections. I'll revert this change!